catalogue/webapp/types/search.ts

```diff
@@ -2,5 +2,5 @@ export type Query = {
   query?: string;
   sortOrder?: string;
   sort?: string;
-  page?: number;
+  page?: string;
```

Comment:
Why has this changed?

Reply:
I saw it as a string since it came from one, but I do notice now it's typed as a number when referenced elsewhere. I'll look at changing and seeing if it fits.

Reply:
OK I reviewed this again. This page comes from `ParsedUrlQuery` (from the [querystring ](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules__types_node_querystring_d_._querystring_.parsedurlquery.html)lib), so it is a string and I'm using it in a string right back, so I thought string made more sense.